src/components/v5/common/TableWithBurgerMenu/TableWithBurgerMenu.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import Table from '~v5/common/Table';
+import { TableWithBurgerMenuProps } from '../Table/types';
+import { useTableWithBurgerMenu } from './hooks';
+
+const displayName = 'v5.common.TableWithBurgerMenu';
+
+const TableWithBurgerMenu = <T,>({
+  className,
+  tableTitle,
+  columns,
+  actions,
+}: TableWithBurgerMenuProps<T>) => {
+  const burgerColumn = useTableWithBurgerMenu(actions);
```

Comment:
It's not a burger, so maybe it would be better to call it actionsColumn or something like this?